### DISCUSSION TITLE
Need for upgrading Microsoft certificates in 2025 in Azure iot device

### DISCUSSION THREAD

**Discussion by anoopkumares** [Azure/azure-iot-sdk-c#2231] [Q&A] [answered]
I am having an iot device which uses azure iot sdk . The device uses the following certificates:

DigiCert Baltimore Root
DigiCert Global Root CA
D-TRUST Root Class 3 CA 2 2009
Recently, I went through the following link.
[](https://baltimore-cybertrust-root.chain-demos.digicert.com/info/index.html)

I saw that one of the certificates used in my firmware application is going to get expired in May 2025.

What problem it will create after expiry date? How can I fix this?

**Comment by ericwolz** [comment, marked as answer]
You need to use the new DigiCert Global G2 CA cert. If you use an expired cert, your device will not be able to connect to the hub.

More info can be found here:

https://techcommunity.microsoft.com/t5/internet-of-things-blog/azure-iot-tls-critical-changes-are-almost-here-and-why-you/ba-p/2393169
